### DISCUSSION TITLE
Super Micro Support?

### DISCUSSION THREAD

**Discussion by KCC-IT-Admin** [netbox-community/devicetype-library#837] [Ideas]
I don't suppose there's any plans to add Super Micro to the list of devices? 

I'm guessing this was deemed low priority due to the sheer number of devices that would need to be added, but it'd be appreciated 

See link for the current Product Matrix - this may be scrapable to convert into a working YAML:  [https://www.supermicro.com/en/products/system/matrix](https://www.supermicro.com/en/products/system/matrix)

**Comment by jeremystretch** [comment]
We'll happily add any accurate, complete device type definitions that people care to write.

**Comment by danner26** [comment]
Hello @KCC-IT-Admin, I can add this to my list of things to start. If you check the repo you will see (at the moment) 4 SuperMicro devices. You are welcome to add any servers you would like in the meantime by submitting a PR, which I would be happy to review
